sdk/servicebus/azure-servicebus/azure/servicebus/_common/auto_lock_renewer.py

```diff
@@ -31,7 +36,7 @@
 )
 
 
-class AutoLockRenewer(object):
+class AutoLockRenewer(object):  # pylint:disable=too-many-instance-attributes
```

Comment:
should async AutoLockRenewer be updated too?

Reply:
async is working differently than the sync one, it relies on the built-in `asyncio.ensure_future` which doesn't have the limitation as the sync ThreadPoolExecutor one